etc/kayobe/environments/ci-builder/stackhpc-ci.yml

```diff
@@ -89,3 +89,6 @@ stackhpc_docker_registry_password: !vault |
 # Username and password of the overcloud host image repository.
 stackhpc_image_repository_username: "{{ stackhpc_docker_registry_username }}"
 stackhpc_image_repository_password: "{{ stackhpc_docker_registry_password }}"
+
+# SELinux set to permissive mode.
+selinux_state: permissive
```

Comment:
set to disabled. If you try to set to permissive, it'll try to reboot the machine and error out